### PR TITLE
Dev ops/csi 1762 make build jobs pass registry credentials

### PR DESCRIPTION
**Pull request by matancarmeli7**

### CODE REVIEW

**Review by tikolsky — APPROVED** [review]

**Review by roysahar-ibm — APPROVED** [review]

**Review by oriyarde — CHANGES_REQUESTED** [review]
@matancarmeli7 most of the issues I see in this PR are the same as the ones I wrote about in https://github.com/IBM/ibm-block-csi-driver/pull/245
please address all remaining conversations there before we continue here.

File: build/ci/jenkins_pipeline_build [hunk 1]
```diff
@@ -48,4 +93,4 @@ pipeline {
         }
 
     }
-}
+}
```
Comment (oriyarde) [inline build/ci/jenkins_pipeline_build:96]: GitHub is telling us that a newline is missing here

Comment (matancarmeli7) [reply build/ci/jenkins_pipeline_build:95]: why do we need a new line between } and }?

Comment (oriyarde) [reply build/ci/jenkins_pipeline_build:98]: we don't. we need a new line between } and the EOF, like [every Posix text file](https://stackoverflow.com/a/729795).

**Review by oriyarde — CHANGES_REQUESTED** [review]
@matancarmeli7 please [finish syncing](https://github.com/IBM/ibm-block-csi-driver/pull/245#discussion_r529654441) the PRs for final approval

**Review by oriyarde — APPROVED** [review]
LGTM

**Pull request merged**